dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -168,7 +169,7 @@ public void setUpTest() {
     attributeService.addTrackedEntityAttribute(filtG);
 
     User user = createUserWithAuth("testUser");
-    user.setTeiSearchOrganisationUnits(Set.of(organisationUnit));
```

Comment:
I assume this was an accident, as this happens to me with Intellij 😁 `Set.of` is easier to read than just `of`.

Reply:
Done